Mathlib/Algebra/Order/Archimedean/Class.lean

```diff
@@ -156,7 +156,8 @@ namespace MulArchimedeanClass
 def mk (a : M) : MulArchimedeanClass M := toAntisymmetrization _ (MulArchimedeanOrder.of a)
 
 /-- An induction principle for `MulArchimedeanClass`. -/
-@[to_additive (attr := elab_as_elim) "An induction principle for `ArchimedeanClass`"]
+@[to_additive (attr := elab_as_elim, induction_eliminator)
```

Comment:
What is `induction_eliminator`?

Reply:
It makes a lemma the default to use when using the `induction` tactic.